### ISSUE TITLE
Couldn't find file /horr/jpn/24.tmd

### ISSUE THREAD

**Issue by RodrigoUnk** [IntelOrca/biorand#807]
### Version

3.2.3

### Seed

R710-NL9M-XNQA00N7ZUY0C07JJKWKM49M5ZZJZQ26ZXZZZZZQ7JWZZZZZZ

### Scenario

RE 1 (Jill)

### Player Character

Claire (Re2 remake)

### What happened?

When going to the cave for the first time, the first door to the right after going down the ladder to enter in the caves is giving me this message and the game crashes (I played with chris the previous night without a problem)
Thanks for BioRand, it's amazing :D

**Comment by RodrigoUnk** [comment]
Could it be related to the fact that I went tio the basement first before going to the guardhouse and then the cave?
With Chris, Rebecca was nowhere to be found after I told her to stay in the mansion (didn't show in the guardhouse, didn't show back in the mansion and was just gone for the rest of the playthrough)
Could the same thing ( or something similar) kinda of have happened with Jill? Because Barry was supposed to be right after that door, and seeing as He was also gone after the mansion (no sight of him in the guardhouse either) so maybe the game didn't know what to do when He was supposed to spawn for the cutscene in that first room in the cave.
Just food for thought :D

**Comment by RodrigoUnk** [comment]
So, just by loading a really old save (with the same seed?) that one room loaded up fine, but just like you said, just by opening the game without the randomizer it was also crashing.
It seems like it is related to the sequence of events that happened in that seed like I mentioned above.
P.S. lçoaded an even older save, and the sequence (I loaded before going to that room, with the first save state that is in the cave)
and by going through the door, the sequence played normally (with Claire RE2 being barry and my Claire RE2R being Jill)
In both saves, the order of the game was maintained ( this was the first time I got the key that forced me to go to the basement earlier to proceed throughout the mansion, leading to the guardhouse, and leading to me going back to the mansion and have nothing changed in there because the game probably thought I was done due to going to the basement earlier or something )
Anyway, thanks for the reply, I am pretty sure you guys will figure it out later :D
